Add_user.css

```diff
@@ -20,52 +27,62 @@ body{
 
 nav{
 	width:100%;
-	background-color: white;
+	background-color: #0C59CF;
```

Comment:
I change the background color to navy blue which is one of the color theme that I decided to apply.